README.md

```diff
@@ -1818,9 +1818,10 @@ Other Style Guides
         }
         break;
       }
-      case 4:
+      case 4: {
```

Comment:
This is intentional - only `case` statements that have block-level variables need braces.